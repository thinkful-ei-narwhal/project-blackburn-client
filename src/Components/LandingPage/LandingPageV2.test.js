import React from 'react';
import { shallow, mount } from 'enzyme'
import toJson from 'enzyme-to-json'
import LandingPageV2 from './LandingpageV2';

describe(`Adds landing page to dom without crashing`, () => {
    it('renders without crashing', () => {
        const wrapper = shallow(
           <LandingPageV2 />
    )
    .find('h1').text('Project Blackburn')
    expect(toJson(wrapper)).toMatchSnapshot()
    })
})