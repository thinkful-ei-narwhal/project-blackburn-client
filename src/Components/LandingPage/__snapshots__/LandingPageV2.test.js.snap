// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Adds dashboard to dom without crashing renders without crashing 1`] = `undefined`;

exports[`Adds landing page to dom without crashing renders without crashing 1`] = `undefined`;
